packages/components/src/card/stories/body.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+/* eslint-disable import/no-extraneous-dependencies */
+import { boolean, text } from '@storybook/addon-knobs';
+/* eslint-enable import/no-extraneous-dependencies */
+
+/**
+ * Internal dependencies
+ */
+import Card from '../index';
+import CardBody from '../body';
+import { getCardStoryProps } from './__utils';
```

Comment:
@gziolo Nope! It was something I did on my end. I can understand the confusion.

I wanted to separate out the Storybook utilities (for knobs) for the Card stories in a file. Storybook's globbing technique loads the stories alphabetically on their left sidebar. Because of that, I renamed `index.js` -> `_index.js` to ensure the main `<Card />` component story got rendered first. To help distinguish the utils stuff from.. non stories, I added double `_`.

I can rename it to just `_util.js` 👍 

(Or if folks have thoughts on another naming convention / location for the file, please lemme know!)

Reply:
What we do with unit tests is we use subfolders to keep there test helpers, fixtures, mocks, etc. This way you don't need to be creative with naming files and there is a nice separation what is the test and what's everything else. We could follow the same pattern here. I haven't tried but in the worst case we will have to update the regular expression which matches stories to ignore subfolders:
https://github.com/WordPress/gutenberg/blob/de715f6d67956cf47e89348bd35cc230cffd6ada/packages/components/storybook/config.js#L18